packages/hierarchy-browser/README.md

```diff
@@ -0,0 +1,146 @@
+# @essex-js-toolkit/hierarchy-browser
+
+A React component to view hierarchical data relating to graph communities.
+
+## Example implementations:
+
+Asynchronous Loading:
+
+```jsx
+const communities = [{communityId: "123", size: 20, neighborSize: 30 },{communityId: "457", size: 10, neighborSize: 5 }]
+
+const loadEntitiesAsync = useCallback(async (params)=>{
+    const data = await handleDataRemotely(params)
+    return ({data, error: undefined})
+},[handleDataRemotely])
+
+const loadNeighborCommunitiesAsync = useCallback(async (params)=>{
+   const data = await handleDataRemotely(params)
+    return ({data, error: undefined})
+},[handleDataRemotely])
+
+<HierarchyBrowser
+    communities={communities}
+    entities={loadEntitiesAsync}
+    neighbors={loadNeighborCommunitiesAsync}
+/>
+```
+
+Static Loading:
+
+```jsx
+const communities = [{communityId: "123", size: 20, entityIds: ["1", "2"], neighborSize: 1 },{communityId: "457", size: 10, entityIds: ["1", "4", "9"], neighborSize: 2 }]
+
+const entities = [{id: "1", attrs:{score: 12.1}},
+{id: "2", attrs:{score: 2.3}},
+{id: "3", attrs:{score: 3.6}},
+{id: "4", attrs:{score: 9.8}},
+{id: "6", attrs:{score: 22.4}},
+{id: "9", attrs:{score: 3.8}}
+] // entities from communities and neighbor communities
+
+const neighbors = [
+    {communityId: "378", size: 1, entityIds: ["6"], connections: 2, edgeCommunityId: "123" },
+    {communityId: "026", size: 1, entityIds: ["3", "6"], connections: 4, edgeCommunityId: "457" }
+]
+
+<HierarchyBrowser
+    communities={communities}
+    entities={entities}
+    neighbors={neighbors}
+/>
+```
+
+## Interfaces
+
+#### Communities
+
+```jsx
+ICommunityDetail[] // only required prop for HierarchyBrowser
```

Comment:
Good JSDocs may be more appropriate for commenting on the interface fields and going into depth on their purpose. Maybe this section should be a birds-eye overview of the interfaces involved.